requirements/pip.txt

```diff
@@ -91,25 +91,25 @@ django-textclassifier==1.0
 nltk==3.4.5
 textblob==0.15.3
 
-django-annoying==0.10.5
+django-annoying==0.10.6
 django-messages-extends==0.6.0
 djangorestframework-jsonp==1.0.2
-django-taggit==1.1.0
+django-taggit==1.2.0
 dj-pagination==2.4.0
 
 # Version comparison stuff
-packaging==19.2
+packaging==20.1
 
 # Commenting stuff
-django-cors-middleware==1.4.0
+django-cors-middleware==1.5.0
 
 # User agent parsing - used for analytics purposes
 user-agents==2.0
 
 # Utilities used to upload build media to cloud storage
-django-storages[azure]==1.7.2
-azure-storage-blob==1.5.0
-azure-storage-common==1.4.2
+django-storages[azure]==1.9
+azure-storage-blob==12.1.0
+azure-storage-common==2.1.0
```

Comment:
We need to be careful with these. We should test,

- build files are uploaded properly
- collectstatic does work